Language/Structure/Control Structure/for.adoc

```diff
@@ -51,7 +51,7 @@ for (initialization; condition; increment) {
 === Example Code
 [source,arduino]
 ----
-// Dim an LED using a PWM pin
+// Brighten an LED using a PWM pin
 int PWMpin = 10;  // LED in series with 470 ohm resistor on pin 10
```

Comment:
```suggestion
int PWMpin = 10;  // LED in series with 470 ohm resistor from pin 10 to ground
```
Whether or not the description is correct depends on the circuit, which was incompletely defined. If the LED is connected between pin 10 and Vcc, then the previous description was correct.